data.json

```
@@ -767,4 +767,10 @@
     "imageUrl": "https://user-images.githubusercontent.com/77366500/115426452-234e8880-a21e-11eb-9e2d-c508acdb921f.jpg",
     "issueNumber": 445
   }
+  {
+    "name": "Mohamed Yaseen Sheriff S",
+    "githubUsername": "Yaseen549",
+    "imageUrl": "https://user-images.githubusercontent.com/43935210/115549892-d377cc00-a2c6-11eb-9b4b-7983603b5c6c.jpg",
+    "issueNumber": 449
+}
```

```suggestion
  }
```
Line this up with the other curly braces